README.md

```diff
@@ -94,54 +94,48 @@ For RabbitMQ
 ```
+### Creating Database Tables:
 
-### Add new user
-
-- To be able to log in add new Employee with your Retro Rabbit email to the TestData.cs
-- Add new Migration for your user
-
-```powershell
-add-migration newMigrationYouCanChooseName
-```
-
-### Run migration
-
-Open Visual Studio 2022 and open the RGO-Server project file. Pull up the nuget package manager console:
-**_Tools_** -> **_NuGet Package Manager_** -> **_Package Manager Console_**
-Make sure the **Default project** is **_RGO.UnitOfWork_**.
-
----
-
-![Image of Package Manager Console](./RGO-UnitOfWork-example.png)
-
+1. Open Visual Studio and go to the package manager.
+2. Change the default project to RR.UnitOfWork.
+3. Run the following commands:
+   
 ```powershell
-Update-Database
+add-migration migrationName
+update-database
 ```
 
-### Checking new user added to the DB you made
-
-- Install **PgAdmin** beforehand. If you locally installed **_PostgreSQL_** be warned that it may interfear with your attempts to connect to the database(Docker).
-- https://www.pgadmin.org/download/pgadmin-4-windows/
+Congratulations! You have now successfully created a database with tables.
 
----
+### Populating Database with Dummy Data:
 
-- Register new RGO server
+1. Make a local copy of the DummyData.sql file In the RR.UnitOfWork Project.
 
-![Register service](./Screenshot%202023-08-02%20173735.png)
+   ![Screenshot 2024-03-12 130755](https://github.com/RetroRabbit/RGO-Server/assets/82169901/178d5ba8-160e-4b28-b280-2b6a08fb02da)
 
----
+3. Copy one of the ``INSERT INTO 'Employee,`` statements in the script.
+   
+   ![Screenshot 2024-03-12 130900](https://github.com/RetroRabbit/RGO-Server/assets/82169901/73545f25-ab5f-4e60-b929-6cd6d0fa781a)
+   
+4. Paste a new INSERT statement and populate it with your information such as your email, name, and surname.
+   
+5. Copy the SQL in the locally created script.
+   
+6. Open PgAdmin, right-click on the RGO database, and select ``Create Script``.
 
-- Update Information and save
+   ![Screenshot 2024-03-12 131219](https://github.com/RetroRabbit/RGO-Server/assets/82169901/5b0fa31a-9e90-4337-8896-fa3d355a5d77)
 
-![Register service - connection](./Screenshot%202023-08-02%20173613.png)
+7. Paste the locally created script in the query screen that pops up.
 
----
+   ![Screenshot 2024-03-12 131332](https://github.com/RetroRabbit/RGO-Server/assets/82169901/87eabaab-4856-4adc-ba54-feb1c6e47512)
 
-- Navigate to Employee table
+8. Click on ``Execute Script``.
+  
+   ![Screenshot 2024-03-12 131356](https://github.com/RetroRabbit/RGO-Server/assets/82169901/69c52269-e074-487e-b489-53ac9c41a5ff)
 
-![PgAdmin Employee Table Navigation](./EmployeeTable-Example.png)
+Once the query is completed successfully, you can go to the employee table and view all rows to see if you have data in the database.
 
-- Check if your new user is added
+Congratulations you have a fully populated database!
 

```

Comment:
```suggestion

### Checking new user added to the DB you made

- Install **PgAdmin** beforehand. If you locally installed **_PostgreSQL_** be warned that it may interfear with your attempts to connect to the database(Docker).

Once the query is completed successfully, you can go to the employee table and view all rows to see if you have data in the database.
```